interface/modules/zend_modules/module/Installer/view/installer/installer/index.phtml

```diff
@@ -154,6 +172,43 @@ $depObj 	= $this->dependencyObject;
 			</div>
 
 	</div>
+
+    <style>
+        .show_hide_log {
+            border-radius: 5px;
+            border: 1px solid #c9c6c6;
+            padding: 5px;
+            text-align: center;
+            cursor: pointer;
+            margin: 5px 5px 0px 0px;
+            color: gray;
+            width: 97%;
+        }
+        .spoiler {
+            display: none;
+            margin-left: 10px;
+            border: 1px solid #c9c6c6;
+            padding: 5px;
+            width: 95%;
+            border-top: 0px;
+        }
+        .modal {
+            display:    none;
+            position:   fixed;
+            z-index:    1000;
+            top:        0;
+            left:       0;
+            height:     100%;
+            width:      100%;
+            background: rgba( 255, 255, 255, .8 )
+            url('http://i.stack.imgur.com/FhHRx.gif')
```

Comment:
Above line seems odd. What is that for?

Reply:
Fixed the path to the gif-loader.
![Screenshot from 2020-01-20 11-21-13](https://user-images.githubusercontent.com/45194408/72714445-63d0d980-3b77-11ea-9187-c9a5ce093f32.png)